code/game/objects/structures/crates_lockers/closets/bodybag.dm

```diff
@@ -142,6 +142,11 @@
 		content.forceMove(B)
 		if(isliving(content))
 			to_chat(content, span_userdanger("You're suddenly forced into a tiny, compressed space!"))
+		if(iscarbon(content))
```

Comment:
If 143 fails this will also fail by default so shouldn’t this be indented?

Reply:
Yeah, I did as you suggested at first, but there's 2 different logics in one loop - one for calculating size class of a bag, and one for telling mob that he is being folded. That is why I decided against it. And this is not performance tight spot, so saving an if and sacrificing clarity is not worth it.

Reply:
yep sounds good for me